build-release-tools/application/releaseNoteConfig.conf

```diff
@@ -0,0 +1,12 @@
+[qurey]
+temp_start_date = 1999-01-01
+temp_end_date = 2100-12-31
+qurey_version = X.Y.Z
+release_version = RELEASE_VERSION
+template_page = TestReleaseNotes
+space_key = RAC1
+rest_url_base = https://rackhd.atlassian.net/wiki/rest/api/content/ 
+[template]
+template_new_row = <tr><td colspan="1"><span>&nbsp;Sprint Release</span></td><td colspan="1">RELEASE_VERSION</td><td colspan="1">2017/06/11</td><td colspan="1"><ac:link><ri:page ri:content-title="RackHD Release Installation Guide" ri:version-at-save="35" /><ac:plain-text-link-body><![CDATA[Installation guide]]></ac:plain-text-link-body></ac:link><span style="color: rgb(0,0,128);"> <ac:image ac:class="emoticon emoticon-yellow-star" ac:alt="(star)"><ri:url ri:value="https://rackhd.atlassian.net/wiki/s/1568075843/6452/18a131a1313f9419e17b00411c0cf7a769a21a5f/_/images/icons/emoticons/star_yellow.png" /></ac:image></span></td><td colspan="1"><ac:link><ri:page ri:content-title="Release Notes X.Y.Z" ri:version-at-save="1" /></ac:link></td><td colspan="1"><a href="https://bintray.com/rackhd/docs/apidoc#files">API doc</a></td><td colspan="1"><ac:link><ri:page ri:content-title="RackHD 3rd party license" ri:version-at-save="11" /><ac:plain-text-link-body><![CDATA[3rd Party Software License Reference]]></ac:plain-text-link-body></ac:link></td></tr>
```

Comment:
i just remind myself about the official release page there may be two tables instead of one 😭